manifests/prometheus/dashboards.d/cell-memory.json

```diff
@@ -0,0 +1,701 @@
+{
+  "dashboard": {
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": "-- Grafana --",
+          "enable": true,
+          "hide": true,
+          "iconColor": "rgba(0, 211, 255, 1)",
+          "name": "Annotations & Alerts",
+          "type": "dashboard"
+        }
+      ]
+    },
+    "editable": true,
+    "gnetId": null,
+    "graphTooltip": 1,
+    "id": null,
+    "iteration": 1666363845398,
+    "links": [],
+    "panels": [
+      {
+        "cards": {
+          "cardPadding": null,
+          "cardRound": null
+        },
+        "color": {
+          "cardColor": "#b4ff00",
+          "colorScale": "sqrt",
+          "colorScheme": "interpolatePlasma",
+          "exponent": 0.5,
+          "mode": "spectrum"
+        },
+        "dataFormat": "timeseries",
+        "datasource": null,
+        "description": "Amount of available memory advertised by rep divided by the actual amount of available memory on cell.\n\nA value much greater than 100% suggests a cell that might be in danger of running out of actual memory and having app memory go into swap or even have apps unexpectedly crash due to machine OOM.\n\nA value much less than 100% suggests a cell that is probably stopping accepting new app instances when it still has plenty of actual memory left. Suboptimal but safe.",
+        "fieldConfig": {
+          "defaults": {},
+          "overrides": []
+        },
+        "gridPos": {
+          "h": 8,
+          "w": 12,
+          "x": 0,
+          "y": 0
+        },
+        "heatmap": {},
+        "hideZeroBuckets": false,
+        "highlightCards": true,
+        "id": 4,
+        "legend": {
+          "show": false
+        },
+        "pluginVersion": "7.5.15",
+        "reverseYBuckets": false,
+        "targets": [
+          {
+            "exemplar": true,
+            "expr": "1e6 * firehose_value_metric_rep_capacity_remaining_memory{bosh_job_name=~\"$bosh_job_name\", bosh_job_ip=~\"$bosh_job_ip\", bosh_job_id=~\"$bosh_job_id\", zone=~\"$zone\"} / on(bosh_job_ip) avg(label_replace(node_memory_MemAvailable_bytes, \"bosh_job_ip\", \"$1\", \"instance\", \"(.*):.*\")) by (bosh_job_ip)",
+            "interval": "",
+            "legendFormat": "",
+            "refId": "A"
+          }
+        ],
+        "timeFrom": null,
+        "timeShift": null,
+        "title": "Cell available memory ratio",
```

Comment:
```suggestion
        "title": "Available memory advertised/Available memory on the cell",
```

Reply:
I'm not sure this is clearer to people who don't expect `/` to mean "divide", instead thinking it'll mean "alternatively".

Reply:
Good point. How about:
> Ratio of available memory: advertised/actual
That way we mention that it is a ratio